libsplinter/src/registry/diesel/mod.rs

```diff
@@ -237,6 +237,54 @@ pub mod tests {
         assert_eq!(node, get_node_1());
     }
 
+    /// Verifies that `update_node` properly updates a node
```

Comment:
Can you add some more details to the commit message? I'm not sure I understand what this change fixes.

Reply:
Added more info to the commit message.